about.html

```diff
@@ -12,53 +12,83 @@
 </head>
 
 <body>
+
+    <div class= "navigation">
+        <img src="img/lambda-black.png" alt="Lambda school logo">
+
+        <nav>
+            <a href="">Home</a>
+            <a href="about.html">About</a>
+            <a href="">Products</a>
+            <a href="">Blog</a>
+            <a href="">Contact</a>
+        </nav>
+    </div>
+
```

Comment:
Great layout of the nav bar. Consider wrapping this entire section in a `<header>` tag for a more precise organization and cleanup should you need it in the future.